doc/source/getting_started.rst

```diff
@@ -0,0 +1,113 @@
+.. _getting-started:
+
+Getting started
+===============
+
+The nomenclature package facilitates working with data templates that follow the format
+developed by the `Integrated Assessment Modeling Consortium (IAMC)
+<https://www.iamconsortium.org>`__. It supports validation of scenario data and region
+processing, which consists of renaming and aggregation of model “native regions” to
+“common regions” used in a project.
+
+There are two main classes that the user interacts with when using the nomenclature
+package, **DataStructureDefinition** and **RegionProcessor**. Additionally, there are a
+number of auxiliary classes which are used by the two main ones to facilitate validation
+and region processing, the two most important ones being **CodeList** and
+**RegionAggregationMapping** (a full list of all classes can be found in :ref:`api`).
+
+A **DataStructureDefinition** contains **CodeLists** for *variables* (including units)
+and *regions* to be used in a model comparison or scenario exercise following the IAMC
+data format.
+
+A **CodeList** is a list of "allowed terms" (or codes), where each term can have several
+attributes (e.g., description, unit, parent region).
+
+A **RegionProcessor** holds a list of RegionAggregationMappings and a
+DataStructureDefinition. This class is used to facilitate region processing for model
+comparison studies.
+
+A **RegionAggregationMapping** is a mapping that defines on a per-model basis how model
+native regions should be renamed and aggregated to comparison regions.
+
```

Comment:
I would structure the classes either in alphabetical order or as you state by importance (first main and then auxiliary). I'd vote for the latter, so first "DataStructureDefinition" and "RegionProcessor" and then the rest.

Reply:
Ah yeah I was debating that, good point, I'll change the order by importance.